### DISCUSSION TITLE
Database Whitelist

### DISCUSSION THREAD

**Discussion by mbos2** [supabase/supabase#352] [Questions] [answered]
Hello.

Does supabase have whitelist? If so, where can I set whitelist for my supabase database?

**Comment by kiwicopple** [comment, marked as answer]
Hey @mbos2 - thanks for the Q&A. We don't have this yet, it's anticipated ~Q1 next year (along with a few other security features).

**Reply by d-mok** [reply]
Hi! I'm looking for this too. I want my supabase API to only accept ajax request from my own domain. Is this feature implemented now?
